### ISSUE TITLE
Not seeing anything in the ribbon

### ISSUE THREAD

**Issue by nonentity1001** [jaumeortola/languagetool-msword10-addin#5]
![1](https://cloud.githubusercontent.com/assets/10053529/19177688/f90452fe-8c96-11e6-8c30-5b49a21757ef.JPG)

Server is running 

![2](https://cloud.githubusercontent.com/assets/10053529/19177824/61a3cf78-8c98-11e6-9716-213be303170b.JPG)

**Comment by jaumeortola** [comment]
Hi, 
The add-in doesn't create its own menu, like "OfficeTab" or "ePUBTools". The LanguageTool buttons should appear in a generic "Add-ins" menu, that is missing in your case. I don't know why.

**Comment by nonentity1001** [comment]
I'll try disabling them - one at a time, Office Tab is a must have for me - I'll get back to you asap

**Comment by nonentity1001** [comment]
OK I figured why the Add-in Ribbon wasn't there - I hadn't ticked the box in Configure Ribbon :doh: 

But I had already uninstalled the langtool add-in via the CP.  Now I can't install it, I get a message saying 'it's already installed', but its not showing in Word or in CP Programs and features.    The uninstall appeared to work OK - it obviously didn't.

So could you tell me what I need to remove so that I can install it again.  I don't want to do a system restore back to yesterday

Thanks

**Comment by jaumeortola** [comment]
Have you tried to remove this file? (Keep a copy) 

C:\Users<user name>\AppData\Roaming\Microsoft\Templates\Normal.dotm

**Comment by nonentity1001** [comment]
Renamed it to SaveNormal.dotm, but that made no difference

![1](https://cloud.githubusercontent.com/assets/10053529/19187269/270241b8-8cd6-11e6-8ab7-f3f74ba690db.JPG)

**Comment by nonentity1001** [comment]
Ok - I found out where it was installed: %localappdata%\Apps\2.0\H511DJ48.6K6\63RAMMAG.5P5.  I renamed it to: %localappdata%\Apps\2.0\H511DJ48.6K6\**JUNK**63RAMMAG.5P5

Then the Install ran OK - and now its in: %localappdata%\Apps\2.0\TQBW50XH.3TT\H47KH6T2.53T

It seems a daft idea to me to be splattering software in App**Data** in the first place, but to put into randomly named directories is bizarre.  I know Google and others put their software in App**Data**, but at least it's in directories with comprehensible names such as: %localappdata%\Google\Chrome

As well as the two Word Add-ins I also use a couple of Excel Add-ins, all of them install into locations such as C:\Program Files (x86)\Toxaris\ePUB Tools.  You might want to consider taking a similar approach.

But the bottom line is that I now have the Add-Ins ribbon, and the Language Tool Buttons are in there.

BTW - there was something else hiding in the Add-Ins ribbon, up until now I've been accessing it via a back door :lol: 

Thanks
